CordovaLib/CordovaLib.xcodeproj/project.pbxproj

```diff
@@ -723,7 +723,7 @@
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
 				VALIDATE_PRODUCT = YES;
-				WK_WEB_VIEW_ONLY = 0;
+				WK_WEB_VIEW_ONLY = 1;
```

Comment:
@bhariharan would make sense to also apply this to the **Cordova.xcconfig** so cocoapod consumers get this out of the box as well?

Reply:
@jhneves Our plugin is consumed through `Cocoapods` out of the box, and they will get this for free. See the config added [here](https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/pull/494/files#diff-53f390d375398624afe1cfe1125f42bfR52). Also, see the `pod` declaration [here](https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/pull/494/files#diff-53f390d375398624afe1cfe1125f42bfR106).